modules/compiler/vecz/test/lit/llvm/emit_memintrinsics.ll

```diff
@@ -92,104 +96,104 @@ declare i64 @__mux_get_local_id(i32)
 
 ; Check if the generated loads and stores are in place
 ; Check the stores for the first memset
-; CHECK: store i64 %ms64val, ptr %sa, align 16
+; CHECK: store i64 %ms64val, ptr %sa, align {{.*}}
```

Comment:
`{{.*}}` at the end of a `CHECK` line never does anything extra since `CHECK` doesn't need to match whole lines.
```suggestion
; CHECK: store i64 %ms64val, ptr %sa
```
and likewise for the rest?